.github/workflows/crib-helm-chart-push-develop.yml

```diff
@@ -0,0 +1,69 @@
+name: Helm Publish
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - "charts/**"
+      - ".github/workflows/crib-helm-chart.yml"
```

Comment:
the new file is called `crib-helm-chart-pr.yml` no?

Reply:
yes, good catch 👍

Reply:
updated